runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h

```diff
@@ -10,10 +10,14 @@
 #include <optional>
 #include <ostream>
 #include <sstream>
+#include <tuple>
```

Comment:
```suggestion
```

Reply:
nah - `std::tie` is in here and i would rather "include what you use" rather than have it transitively included through llvm headers (eventually i'll connect [iwyu](https://github.com/include-what-you-use/include-what-you-use))